llvm/include/llvm/AsmParser/AsmParserState.h

```diff
@@ -0,0 +1,17 @@
+#ifndef LLVM_ASMPARSER_ASMPARSER_STATE_H
+#define LLVM_ASMPARSER_ASMPARSER_STATE_H
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/Support/SMLoc.h"
+#include "llvm/IR/Value.h"
+#include <cstddef>
+
+namespace llvm {
+struct AsmParserState {
+  DenseMap<Function*, FileLocRange> functions;
```

Comment:
```suggestion
  DenseMap<Function*, FileLocRange> Functions;
```
Nit. Here and everywhere else. LLVM code convention asks one to start variable names with Capital Letter.